src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -173,16 +237,13 @@ public static VisionIO[] createCameras() {
    * @return The array of vision IOs.
    */
   public static VisionIO[] createSimCameras() {
-    return new VisionIO[] {
-      new VisionIOSim("LeftCam", "LeftSystem"), // TODO
-      new VisionIOSim("RightCam", "RightSystem")
-    };
+    return new VisionIO[] {new VisionIOSim(leftCam), new VisionIOSim(rightCam)};
   }
 
   public void periodic() {
     for (var camera : cameras) {
-      camera.io.updateInputs(camera.inputs, new Pose3d(pose)); // TODO so sketch
-      Logger.processInputs("Vision", camera.inputs); // TODO so sketch
+      camera.updateInputs(new Pose3d(pose));
+      Logger.processInputs("Vision", camera.inputs);
```

Comment:
probably need each camera to have a different name for the log, could just use the camera name

Reply:
or follow modules example and handle the Logger call in Vision.updateInputs